cmd/login.go

```diff
@@ -81,7 +81,7 @@ var loginCmd = &cobra.Command{
 
 func init() {
 	RootCmd.AddCommand(loginCmd)
-	loginCmd.Flags().StringVarP(&controllerURL, "controllerURL", "s", "http://localhost:8080", "Controllers URL")
+	loginCmd.Flags().StringVarP(&controllerURL, "controllerURL", "s", "http://localhost:80", "Controllers URL")
```

Comment:
That is where nginx is running. We thought that would be easier, from the user perspective, to access on port 80.